### DISCUSSION TITLE
VLC issues with 500

### DISCUSSION THREAD

**Discussion by skijohn877** [GerryDazoo/Slinger#602] [General]
I have a 500 that seems to having issues with HD streaming with VLC. Works fine with streaming using Slinger, but with VLC starts displays video for a second then terminates. I recall seeing something about a restart/reconnect parameter that was mentioned is a previous post, was that parameter ever implemented? Running version 4.01 python code on raspberry pi 3b. Thanks

**Comment by tryin100** [comment]
try turning on 'loop 1' on vlc.  that's my only suggestion
